internal/provider/resource_time_sleep_test.go

```diff
@@ -30,7 +30,7 @@ func TestResourceTimeSleepCreate(t *testing.T) {
 		t.Fatalf("unable to parse test duration: %s", err)
 	}
 
-	sleepResource := NewTimeSleepResource()
+	sleepResource := NewTimeSleepTestResource()
```

Comment:
Rather than having a special test resource here, we should just update these tests to call `Configure`. We should try to keep the tests as close to real world expectations as possible 👍🏻